topics/eslint/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Nicholas C. Zakas
+display_name: ESLint
+github_url: https://github.com/eslint/eslint
+logo: eslint.png
+related: eslint-plugin, eslint-config
+released: June 2013
+short_description: The pluggable linting utility for JavaScript and JSX
+topic: eslint
+url: https://eslint.org/
+---
+ESLint is an extensible static-analysis tool for JavaScript and related languages that helps you catch errors before they bite you in production.
```

Comment:
I like the `before they bite you` part :smile: but it feels a bit casual for a referential prose style. We also try to avoid "you" references.

Could you rephrase? Actually, even `helps you catch errors before they bite` would be 👍 by me.

Reply:
How does the new one sound?